README.md

```diff
@@ -4,6 +4,8 @@
 
 Binding generator for [FFI](https://dart.dev/guides/libraries/c-interop) bindings.
 
+> Note: ffigen only supports parsing `C` headers at the moment.
```

Comment:
Leave out the "at the moment".

We will never support C++ headers, because C++ does not have a stable ABI like C (or use of `extern "C"` in C++ headers). Without a stable ABI it is impossible to know in what register/stack locations to pass arguments and this will lead to endless segmentation faults.

We could potentially support C++ headers which have `extern "C"` on every symbol, but we will never support C++ headers.

Reply:
Done.